### PR TITLE
Add tags in toc.yml for subcommand merge

### PR DESCRIPTION
**Pull request by qinezh**
@vwxyzh @vicancy @superyyrrzz @hellosnow @ansyral @chenkennt

### CODE REVIEW

File: src/docfx/SubCommands/MetadataMerger.cs [hunk 1]
```diff
@@ -104,6 +109,53 @@ private static void MergeToc(MetadataMergeParameters parameters, string outputBa
...
+            if (content != null)
+            {
+                var metadata = content.Metadata;
+                object tagsObj;
+                if (metadata != null && metadata.TryGetValue("tags", out tagsObj))
+                {
+                    var tags = tagsObj as object[];
+                    if (tags != null)
+                    {
+                        var tagsList = tags.Select(tag => tag as string).Where(tag => tag != null).ToArray();
```
Comment (ansyral) [inline src/docfx/SubCommands/MetadataMerger.cs]: >Select(tag => tag as string).Where(tag => tag != null) [](start = 44, length = 54)

.Where(tag => tag is string) ?

Comment (qinezh) [inline src/docfx/SubCommands/MetadataMerger.cs]: in that way, tagsList would be a list of object instead of a list of string

**Comment by qinezh** [comment]
Closed as the design is changed.

File: src/docfx/SubCommands/MetadataMerger.cs (hunk 1 above)
Comment (ansyral) [inline src/docfx/SubCommands/MetadataMerger.cs]: i see

---
In reply to: [89735853](https://github.com/dotnet/docfx/pull/931#discussion_r89735853) [](ancestors = 89735853)

**Pull request closed without merging**